VAMobile/src/components/ClickForActionLinkDeprecated.tsx

```diff
@@ -207,11 +206,9 @@ const ClickForActionLinkDeprecated: FC<LinkButtonProps> = ({
   return (
     <TouchableWithoutFeedback testID={testID} {...pressableProps}>
       <Box flexDirection={'row'} alignItems={'center'} py={py} pr={pr}>
-        {!hideIcon && (
-          <Box pr={3}>
-            {linkType ? (
-              <Icon name={getIconName()} fill={iconColorOverride ?? theme.colors.icon.link} width={25} height={25} />
-            ) : undefined}
+        {!hideIcon && linkType && (
```

Comment:
This is the deprecated ClickForActionLink, does this need to be updated in the component library?